CondCore/CondDB/interface/IOVProxy.h

```diff
@@ -122,8 +122,8 @@ namespace cond {
       // otherwise, a new query will be executed using the resolved group boundaries.
       // throws if the target time cannot be found.
       cond::Iov_t getInterval(cond::Time_t time);
-      cond::Iov_t getInterval(cond::Time_t time, cond::Time_t defaultIovSize);
 
+      //
```

Comment:
was this meant to be a beginning of a comment?

Reply:
not sure :-) - removed now